opensearch/src/main/java/org/opensearch/sql/opensearch/request/OpenSearchQueryRequest.java

```diff
@@ -101,6 +103,21 @@ public OpenSearchQueryRequest(
     this.sourceBuilder = sourceBuilder;
     this.exprValueFactory = factory;
     this.includes = includes;
+    this.fieldTypeTolerance = false;
```

Comment:
fieldTypeTolerance should not associate with OpenSearchRequest, It is been used when OpenSearchExprValueFactory parse response.?

Reply:
It is only here so that it can be passed down to the OpenSearchResponse. It is not used directly in OpenSearchRequest.

Reply:
Updated.